sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryParallelReadStreamReader.java

```diff
@@ -35,9 +35,9 @@
 /**
  * Iterates over all rows assigned to a particular reader in a read session.
  */
-class BigQueryParallelReader<T> extends BoundedSource.BoundedReader<T> {
+class BigQueryParallelReadStreamReader<T> extends BoundedSource.BoundedReader<T> {
```

Comment:
Why do we decide to change the name?

Reply:
In these changes, BigQueryParallelReadStreamReader is coupled tightly with BigQueryParallelReadStreamSource -- in fact, I'm not sure it's correct stylistically for the Reader to exist outside the context of the StreamSource. I changed the name to reflect this coupling.

Reply:
I didn't get the whole picture...